src/platform/forms-system/src/js/utilities/validations/index.js

```diff
@@ -136,3 +136,25 @@ export function isValidPartialMonthYearInPast(month, year) {
       momentDate.isSameOrBefore(moment().startOf('month')))
   );
 }
+
+/**
+ * Check validations for Custom pages
+ * @param {Function[]} validations - array of validation functions
+ * @param {*} data - field data passed to the validation function
+ * @param {*} fullData - full and appStateData passed to validation function
+ * @returns {String[]} - error messages
+ */
+export const checkValidations = (
```

Comment:
Since this method returns error messages, I think it's name could reflect that. I liked your `getValidationErrors` naming in `src/platform/forms-system/src/js/utilities/data/profile.js`.

Reply:
I like this idea!